build.gradle

```diff
@@ -24,18 +24,9 @@ ext {
 }
 
 dependencies {
-    testImplementation("org.testng:testng:7.5") {
-        //exclude to pass compilation step due to conflicting versions in the path
-        exclude group: "org.slf4j", module: "slf4j-log4j12"
-    }
+    testImplementation("org.testng:testng:7.5")
     testImplementation("jmock:jmock:1.+")
-    testImplementation("org.apache.directory.server:apacheds-all:1.5.7")
```

Comment:
Surprising that this can be removed with no impact. Did you happen to notice if the related code was removed/disabled?